info/buildSrc/src/main/kotlin/org/cqfn/diktat/generation/docs/GenerationDocs.kt

```diff
@@ -220,3 +222,5 @@ private fun findBoldOrItalicText(regex: Regex,
     }
     return correctedLine
 }
+
+private fun String.getFirstNumber() = String(this.replace("\\s+".toRegex(), "").toCharArray().takeWhile { it.isDigit() || it == '.' }.toCharArray())
```

Comment:
```suggestion
private fun String.getFirstNumber() = replace("\\s+".toRegex(), "").takeWhile { it.isDigit() || it == '.' }
```
Explicitly constructing String from CharArray is not needed here, I believe.